config/indexer.go

```diff
@@ -16,6 +16,9 @@ type Indexer struct {
 	// individual multihashes within a Put. A value of 1 means no concurrency,
 	// and zero uses the default.
 	CorePutConcurrency int
+	// DHStoreURL is the base URL for the DHStore service. This option value
+	// tells the indexer core to use a DHStore service, if configured.
+	DHStoreURL string
```

Comment:
Can we also add an ability to disable storing to the local store and use only dhstore?